pocs/tests/test_camera.py

```diff
@@ -297,14 +301,18 @@ def test_exposure_not_connected(camera):
     camera._connected = True
 
 
-def test_observation(camera):
+def test_observation(camera, images_dir):
     """
     Tests functionality of take_observation()
     """
     field = Field('Test Observation', '20h00m43.7135s +22d42m39.0645s')
     observation = Observation(field, exp_time=1.5 * u.second)
+    observation.seq_time = 'seq_time'
```

Comment:
I don't think we rely on any pattern match anywhere for the seq_time but this seems a little odd to just call it 'seq_time'. Why not a dummy pattern that looks like a datestamp?

Reply:
Good point.